packages/firestore/src/lite-api/aggregate.ts

```diff
@@ -15,40 +15,95 @@
  * limitations under the License.
  */
 
-import { DocumentData, Query, queryEqual } from './reference';
+import { Value } from '../protos/firestore_proto_api';
+import { invokeRunAggregationQueryRpc } from '../remote/datastore';
+import { hardAssert } from '../util/assert';
+import { cast } from '../util/input_validation';
 
-export class AggregateQuery<T = DocumentData> {
+import { getDatastore } from './components';
+import { Firestore } from './database';
+import { Query, queryEqual } from './reference';
+import { LiteUserDataWriter } from './reference_impl';
+
+/**
+ * An `AggregateQuery` computes some aggregation statistics from the result set of
+ * a base `Query`.
+ */
+export class AggregateQuery {
   readonly type = 'AggregateQuery';
-  readonly query: Query<T>;
+  /**
+   * The query on which you called `countQuery` in order to get this `AggregateQuery`.
+   * Query type is set to unknown to avoid error caused by query type converter.
```

Comment:
Please remove "Query type is set to unknown to avoid error caused by query type converter..."

Reply:
did I keep missing this comment? I thought i removed it.